zap/src/main/java/org/zaproxy/zap/extension/pscan/ExtensionPassiveScan.java

```diff
@@ -310,20 +309,6 @@ protected PassiveScannerList getPassiveScannerList() {
 
             // Read from the configs
             scannerList.setAutoTagScanners(getPassiveScanParam().getAutoTagScanners());
-
-            // Load the  'switchable' plugins
-            List<PluginPassiveScanner> listTest =
-                    List.of(new RegexAutoTagScanner(), new StatsPassiveScanner());
```

Comment:
Regex is next?

Reply:
Okay 😃

Reply:
Yup I see now, it's in the list but then ignored in the loop.

Reply:
Should we deprecate that too?

Reply:
Not yet, that will have to be moved along other classes first.